fluentui_core/src/main/java/com/microsoft/fluentui/theme/token/controlTokens/TabsTokens.kt

```diff
@@ -0,0 +1,58 @@
+package com.microsoft.fluentui.theme.token.controlTokens
+
+import android.os.Parcelable
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+import com.microsoft.fluentui.theme.FluentTheme
+import com.microsoft.fluentui.theme.ThemeMode
+import com.microsoft.fluentui.theme.token.AliasTokens
+import com.microsoft.fluentui.theme.token.ControlInfo
+import com.microsoft.fluentui.theme.token.FluentColor
+import kotlinx.parcelize.Parcelize
+
+data class TabsInfo(
+    val style: PillButtonStyle = PillButtonStyle.Neutral
+) : ControlInfo
+
+@Parcelize
+open class TabsTokens : PillBarTokens(), Parcelable {
```

Comment:
Add prefix of "Pill" in tab as there would be rail & bar variant as well.

Reply:
Done